libtransmission/watchdir.h

```diff
@@ -60,6 +60,7 @@ class Watchdir
         std::chrono::milliseconds rescan_interval = generic_rescan_interval_);
 
 private:
+    // NOLINTNEXTLINE(readability-identifier-naming)
     static inline auto generic_rescan_interval_ = std::chrono::milliseconds{ 1000 };
```

Comment:
The other two (at least) static private non-const occurrences I saw before were fixes, this one is ignored, why? Do you want to adjust the configuration in any way?

Reply:
7d959a60c3c72cc3df6ac6ba8ece07e83d4a624f I don't remember why I did not ignore the other 2 occurences.

> Do you want to adjust the configuration in any way?

Yes, I want to tighten the variable naming conventions in libtransmission in a future PR. For example by setting `readability-identifier-naming.PrivateMemberSuffix` to `_`.

This will line-up libtransmission with other subfolders such as qt and gtk.